src/dealWizard/stages/primaryDao/primaryDao.html

```diff
@@ -0,0 +1,37 @@
+<template>
+  <div class="page animated-page stageContainer stage1 au-animate">
+    <div class="headingContainer">
+      <h1 class="heading heading1 gradient">Token swap proposal</h1>
+      <hr />
+    </div>
+    <div class="infoContainer">
+      <div class="sideText">
+        <h2 class="heading heading3 title">Primary DAO</h2>
+        Enter the Primary DAO information
+      </div>
+      <div class="form">
+        <div class="infoInput">
+          <div class="labelContainer">
+            <label for="proposalName">DAO Name</label>
+            <span>
+              ${wizard.wizardResult.daos[0].name.length}/50
+            </span>
+          </div>
+          <input id="proposalName" value.bind="wizard.wizardResult.daos[0].name" maxlength="50" />
+          <span>${errors.name}</span>
+        </div>
+        <div class="buttonContainer">
+          <pbutton type="secondary" click.delegate="wizardService.previous(wizardManager)">
```

Comment:
I am tempted to suggest we turn these buttons into reusable components where we just pass in the wizardService.

Maybe could have a component that contains all of them and displays the exact required buttons for first, middle and last stages.